decidim-templates/config/i18n-tasks.yml

```diff
@@ -0,0 +1,10 @@
+---
```

Comment:
Shouldn't we have this ignores in the root `config/i18n-tasks.yml`?
Is this file even taken into account by CI?

Reply:
Truth is I'm not sure why I included this file, I guess I copied it from the `decidim-elections` module and never reviewed its use. I'll remove it and we can add it back later if we find a reason.